mno_analysis_tools/run-mno-analysis.sh

```diff
@@ -32,11 +41,8 @@ echo "Starting run with id '$RUN_ID'"
 
 ./docker-run-fetch-raw-messages.sh --profile-memory ./data "$DOMAIN" "$TOKEN" "$OUTPUT_DIR"
 
-./docker-run-compute-window-of-downtime.sh --profile-memory ./data "$RAW_MESSAGES_FILE_PATH" \
+./docker-run-compute-window-of-downtime.sh --profile-memory ./data "${OUTPUT_DIR%/}/raw_messages.json" \
```

Comment:
Is the %/ necessary?

Reply:
It will be used to remove trailing slash if included in output dir arg, i.e, if output dir is ~/avf_data/mno_data/ it will remove the trailing slash to prevent the path being with double slash like, ~/avf_data/mno_data//raw_messages.json